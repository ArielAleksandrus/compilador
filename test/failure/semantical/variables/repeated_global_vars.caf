int a, a = a + 3;

programa{}