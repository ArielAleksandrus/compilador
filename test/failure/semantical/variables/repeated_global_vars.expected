ERROR => Line 1, col 8: Redeclaration of variable. 'a' previously found in line 1, col 5
